src/compile-cache.js

```diff
@@ -99,7 +99,7 @@ function writeCachedJavascript (relativeCachePath, code) {
 
 function addSourceURL (jsCode, filePath) {
```

Comment:
Every one of our compilers already adds the source map and does the file:// work on Windows. I suspect this entire function can be removed.